packages/core/src/codegen/service.ts

```diff
@@ -88,22 +86,4 @@ export class CodegenService extends Emittery {
       msg: `Wrote new file at generated/index.ts`,
     });
   }
-
-  generateSchemaFile({ schema }: { schema: Schema }) {
```

Comment:
I hadn't thought about this, but I kinda agree that we don't need to write the `generated/schema.graphql` file. I thought that it would be useful for stuff like `graphql-codegen`, but I'm not sure if anyone's using it.

I polled the telegram.

Reply:
From what I remember you can use` graphql-codegen` with just the endpoint. If that's not the case then I think we should include it because I think that is pretty useful.